src/lib/convenience/RegisterProviderRequest.h

```diff
@@ -50,7 +50,7 @@ typedef struct RegisterProviderRequest
   RegisterProviderRequest();
 
   std::string  render(Format format, std::string indent);
-  std::string  check(RequestType requestType, Format format, std::string indent, std::string preError, int counter);
+  std::string  check(ConnectionInfo* ciP, RequestType requestType, Format format, std::string indent, std::string preError, int counter);
```

Comment:
I don't know the issue number, but I can look it up.
However, that issue is most probably about making **all** check methods and render methods having the same signature and we are still far from that goal.
The issue would not be closed.